src/UIComponents/ModelCard.jsx

```diff
@@ -45,52 +45,63 @@ class ModelCard extends Component {
         <div style={styles.modelCardContainer}>
           <h3 style={styles.modelCardHeader}>{trainedModelDetails.name}</h3>
           <div style={styles.modelCardSubpanel}>
-            <h5 style={styles.modelCardHeading}>Summary</h5>
+            <h5 style={styles.modelCardHeading}>Accuracy</h5>
             <div style={styles.modelCardContent}>
               <p style={styles.modelCardDetails}>
-                Predict {label.id} based on{" "}
-                {selectedFeatures.length > 0 && percentCorrect && (
+                {percentCorrect && (
                   <span>
-                    {selectedFeatures.join(", ")} with {percentCorrect}%
-                    accuracy.
+                    {percentCorrect}%
                   </span>
                 )}
               </p>
             </div>
           </div>
           <div style={styles.modelCardSubpanel}>
-            <h5 style={styles.modelCardHeading}>About the Data </h5>
+            <h5 style={styles.modelCardHeading}>Intended Use</h5>
             <div style={styles.modelCardContent}>
-              {datasetDetails.description && (
-                <p style={styles.modelCardDetails}>
-                  {datasetDetails.description}
-                </p>
-              )}
-              {datasetDetails.numRows && (
+              {trainedModelDetails.potentialUses && (
                 <p style={styles.modelCardDetails}>
-                  Dataset size: {datasetDetails.numRows} rows
+                  {trainedModelDetails.potentialUses}
                 </p>
               )}
             </div>
           </div>
           <div style={styles.modelCardSubpanel}>
-            <h5 style={styles.modelCardHeading}>Intended Use</h5>
+            <h5 style={styles.modelCardHeading}>Limitations and Warnings</h5>
             <div style={styles.modelCardContent}>
-              {trainedModelDetails.potentialUses && (
+              {trainedModelDetails.potentialMisuses && (
                 <p style={styles.modelCardDetails}>
-                  {trainedModelDetails.potentialUses}
+                  {trainedModelDetails.potentialMisuses}
                 </p>
               )}
             </div>
           </div>
           <div style={styles.modelCardSubpanel}>
-            <h5 style={styles.modelCardHeading}>Warnings</h5>
+            <h5 style={styles.modelCardHeading}>About the Data</h5>
             <div style={styles.modelCardContent}>
-              {trainedModelDetails.potentialMisuses && (
+              {datasetDetails.description && (
                 <p style={styles.modelCardDetails}>
-                  {trainedModelDetails.potentialMisuses}
+                  {datasetDetails.description}
                 </p>
               )}
+              {datasetDetails.numRows && (
```

Comment:
Similarly here, what if there are `0` rows?

Reply:
There's no way to successfully train a model and get to the summary screen if there are 0 rows of data so I'm not too worried about this case.